src/main/java/net/uaznia/lukanus/hudson/plugins/gitparameter/GitParameterDefinition.java

```diff
@@ -424,6 +443,21 @@ public GitSCM getProjectSCM(AbstractProject<?, ?> project) {
             if (projectScm instanceof GitSCM) {
                 return (GitSCM) projectScm;
             }
+
```

Comment:
Invert condition REPO_SCM.equals(projectScm.getClass().getName()) and extract all condition to a separate method (eg isRepoScmProject(SCM porojectScm))

Extract try to seperate method